src/code-editor/background-inline-svg.scss

```diff
@@ -16,15 +16,13 @@
     @if ($selector == ':root') {
       @include _themed-background-inline-svg($path, $tokens, $stroke, $fill, 'light');
       // dark mode is not supported in browsers that do not support css variables
-      /* stylelint-disable-next-line plugin/no-unsupported-browser-features */
       @supports (--css-variable-support-check: #000) {
```

Comment:
There is an opportunity to remove this check completely, because browsers without CSS vars are completely unsupported because other features